### PR TITLE
Change rayyan-store connection test to /liveness

### PR DESCRIPTION
**Pull request by fabubaker**

### CODE REVIEW

File: helm/rayyan/templates/tests/test-connection.yaml [hunk 1]
```diff
@@ -63,7 +63,7 @@ spec:
     - name: wget
       image: busybox
       command: ['wget']
-      args:  ['http://rayyan-store0:3000/ping']
+      args:  ['http://rayyan-store0:3000/liveness']
```
Comment (hammady) [inline helm/rayyan/templates/tests/test-connection.yaml]: What is the best test here, liveness or readiness?

Comment (fabubaker) [reply helm/rayyan/templates/tests/test-connection.yaml]: I think liveness? These tests are to ensure that the pod is up and running. Handling requests is out of scope I think.

Comment (hammady) [reply helm/rayyan/templates/tests/test-connection.yaml]: I'm wondering why we need tests then. The `--wait` flag in the `helm upgrade` command will not return until all pods are in `Ready` state, i.e. the `readinessProbe` is succeeding. What does it mean to do a liveness afterwards?

Comment (fabubaker) [reply helm/rayyan/templates/tests/test-connection.yaml]: > What does it mean to do a liveness afterwards?

The one use case I can think of is if we were to skip the `--wait` flag for whatever reason, the tests would come in handy.

But assuming we keep `--wait`, even checking readiness in the tests afterwards is redundant no?

Comment (hammady) [reply helm/rayyan/templates/tests/test-connection.yaml]: Even if it is redundant, liveness is only a subset of readiness. I'm starting to think that tests are useless if we have proper `readinessProbes` for all pods and `--wait`.

Comment (fabubaker) [reply helm/rayyan/templates/tests/test-connection.yaml]: I see your point, but for the purposes of getting the `readinessProbe` to prod, what should I do here?

**Pull request merged**